### PR TITLE
chore(ci): Fix golangci-lint

### PR DESCRIPTION
**Pull request by yevgenypats**

### CODE REVIEW

**Review by erezrokah — APPROVED** [review]

**Comment by hermanschaaf** [comment]
We'll need to apply this fix to the plugins in https://github.com/cloudquery/cloudquery/pull/2141 as well

File: .github/workflows/dest_postgresql.yml [hunk 1]
```diff
@@ -45,7 +45,8 @@ jobs:
         uses: golangci/golangci-lint-action@v3
         with:
           version: v1.49.0
-          args: "--config ${GITHUB_WORKSPACE}/plugins/.golangci.yml"
+          working-directory: plugins/destination/postgresql
+          args: "--config ../.golangci.yml"
```
Comment (hermanschaaf) [inline .github/workflows/dest_postgresql.yml:49]: Looks like it's got to be `../../.golangci.yml`

Comment (erezrokah) [inline .github/workflows/dest_postgresql.yml]:
```suggestion
          args: "--config ../../.golangci.yml"
```

It should be two levels up

**Review by hermanschaaf — APPROVED** [review]

**Comment by erezrokah** [comment]
>We'll need to apply this fix to the plugins in https://github.com/cloudquery/cloudquery/pull/2141 as well

👍 I'll add this in a minute

**Pull request merged**